target/linux/ramips/image/mt7621.mk

```diff
@@ -286,6 +286,22 @@ define Device/arcadyan_we420223-99
 endef
 TARGET_DEVICES += arcadyan_we420223-99
 
+define Device/zte_mf286c3
+  $(Device/nand)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  VID_HDR_OFFSET := 2048
+  IMAGE_SIZE := 41984k
+  KERNEL_SIZE := 5120k
+  UBINIZE_OPTS := -E 5
```

Comment:
This lines can be dropped since you add `$(Device/nand)`:
```
  BLOCKSIZE := 128k
  KERNEL_SIZE := 4096k
  PAGESIZE := 2048
  UBINIZE_OPTS := -E 5
```